app/views/user_mailer/invitation.text.erb

```diff
@@ -7,11 +7,13 @@
 <%= t ".been_invited" %>
 <% end %>
 
-<%= t ".part1" %>
+<%= t ".text.ao3_intro" %>
```

Comment:
We'd prefer to pass the OTW website URL as a variable here instead of writing it out in the yml file. That way, if it ever changes, we don't have to edit all the languages... and even if it never changes, we don't have to worry about any typos sneaking in.

Reply:
Do you know where I should be defining the OTW website URL as a variable/how to find if it's already been defined as a variable somewhere?

Reply:
Oh, sorry, I meant a [variable for translation](https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations), and not, e.g., a constant in the config file. So you'd define it here, like 

`<%= t(".text.about", otw_url: "https://transformativeworks.org") %>`

and then use it in the definition in the yml file like

```
invitation:
  text:
    about: "The Archive of Our Own (AO3) is a free, noncommercial archive built by and for fans. Our servers are owned by our parent nonprofit, the Organization for Transformative Works (%{otw_url}), which works to protect fan rights and preserve fanworks. We welcome all kinds of fanworks, including fanfiction, fanart, fanvids, and podfic from any fandom."
```